types/protocol/protocol.go

```diff
@@ -1,3 +1,4 @@
+// Package protocol includes the "API" of events the rule-engine can consume. All producers who intend to add support for tracee need to support this protocol.
```

Comment:
Usually you document packages in Go by adding the `doc.go` file? See https://go.dev/blog/godoc

Reply:
From what I read here, it seems `doc.go` is for large packages (for example the `tracee` package, they use `encoding/gob` as theirs). For small ones it's suggested to do short comments like the one above so I think this is fine.

Reply:
Okey, I guess protocol won't become huge package and we can resolve this conversation for now.